### ISSUE TITLE
The Browser library does not work, perhaps missing dependencies

### ISSUE THREAD

**Issue by makimat** [ppodgorsek/docker-robot-framework#372]
**Describe the bug**
A simple test using the Browser library fails to execute due to errors with the built in chrome browser inside node packages.

**To Reproduce**
1. Build the latest master (33d9623) locally: `docker build -t robot-baseimage .`
2. Run a simple test with newly built image: ``docker run --shm-size=1g --rm -it -v "`pwd`/browsertest:/opt/robotframework/tests" -v "`pwd`/reports:/opt/robotframework/reports" -e "ROBOT_OPTIONS=--loglevel DEBUG" robot-baseimage``
3. Check the output and log

**Expected behavior**
Test run succeeds without errors.

**Robot template**
Test set: [test.robot.txt](https://github.com/ppodgorsek/docker-robot-framework/files/7126146/test.robot.txt)

**Screenshots**

**Please provide any relevant information, such as:**
 - OS: Windows 10, Docker Desktop (engine v20.10.7)
 - Browser: Both chrome and firefox

**Additional context**
Full robot log file from the failing test run: [output.xml.txt](https://github.com/ppodgorsek/docker-robot-framework/files/7126141/output.xml.txt)

Note especially these errors in the output:
```
[pid=64][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-907428/chrome-linux/chrome: /usr/lib/libasound.so.2: no version information available (requir
ed by /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-907428/chrome-linux/chrome)
[pid=64][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-907428/chrome-linux/chrome: /usr/lib/libasound.so.2: no version information available (requir
ed by /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-907428/chrome-linux/chrome)
[pid=64][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-907428/chrome-linux/chrome: Relink `/usr/lib/libbz2.so.1' with `/usr/glibc-compat/lib/libc.so
.6' for IFUNC symbol `strcat'
```

**Comment by ssallmen** [comment]
I bumped into the same issue when trying to run the sample Browser Library tests in repo using chromium or webkit. Firefox seems to work ok with those. I get this error when using webkit as the browser:
```
==============================================================================
Browser-Library
==============================================================================
Visit Google                                                          [ WARN ] Keyword 'Take Screenshot' could not be run on failure:
Error: Tried to take screenshot, but no page was open.
| FAIL |
Error: browserType.launch: Failed to launch: Error: spawn /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/webkit-1516/pw_run.sh ENOENT
=========================== logs ===========================
<launching> /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/webkit-1516/pw_run.sh --inspector-pipe --headless --no-startup-window
[pid=N/A] starting temporary directories cleanup
[pid=N/A] finished temporary directories cleanup
============================================================
------------------------------------------------------------------------------
Browser-Library                                                       | FAIL |
1 test, 0 passed, 1 failed
==============================================================================
```

And using chromium as the browser I get this:
```
==============================================================================
Browser-Library
==============================================================================
Visit Google                                                          [ WARN ] Keyword 'Take Screenshot' could not be run on failure:
Error: Tried to take screenshot, but no page was open.
| FAIL |
Error: browserType.launch: Protocol error (Browser.getVersion): Browser closed.
==================== Browser output: ====================
<launching> /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome --disable-background-networking --enable-features=NetworkService,NetworkServiceInProcess --disable-background-timer-throttling --disable-backgrounding-occluded-windows --disable-breakpad --disable-client-side-phishing-detection --disable-component-extensions-with-background-pages --disable-default-apps --disable-dev-shm-usage --disable-extensions --disable-features=TranslateUI,BlinkGenPropertyTrees,ImprovedCookieControls,SameSiteByDefaultCookies,LazyFrameLoading,GlobalMediaControls,DestroyProfileOnBrowserClose --allow-pre-commit-input --disable-hang-monitor --disable-ipc-flooding-protection --disable-popup-blocking --disable-prompt-on-repost --disable-renderer-backgrounding --disable-sync --force-color-profile=srgb --metrics-recording-only --no-first-run --enable-automation --password-store=basic --use-mock-keychain --no-service-autorun --user-data-dir=/tmp/playwright_chromiumdev_profile-EipeDn --remote-debugging-pipe --headless --hide-scrollbars --mute-audio --blink-settings=primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4 --no-sandbox --no-startup-window
    [ Message content over the limit has been removed. ]
...bars --mute-audio --blink-settings=primaryHoverType=2,availableHoverTypes=2,primaryPointerType=4,availablePointerTypes=4 --no-sandbox --no-startup-window
<launched> pid=68
[pid=68][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome: /usr/lib/libasound.so.2: no version information available (required by /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome)
[pid=68][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome: /usr/lib/libasound.so.2: no version information available (required by /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome)
[pid=68][err] /usr/local/lib/python3.9/site-packages/Browser/wrapper/node_modules/playwright/.local-browsers/chromium-901522/chrome-linux/chrome: Relink `/usr/lib/libbz2.so.1' with `/usr/glibc-compat/lib/libc.so.6' for IFUNC symbol `strcat'
[pid=68] <process did exit: exitCode=null, signal=SIGSEGV>
[pid=68] starting temporary directories cleanup
============================================================
------------------------------------------------------------------------------
Browser-Library                                                       | FAIL |
1 test, 0 passed, 1 failed
==============================================================================
```

@ppodgorsek, could this be related to the glibc library usage that Playwright guys are saying is restricting using alpine linux as the basis for Playwright? Check https://playwright.dev/docs/docker#alpine

I have been trying to build a docker image for using Browser Library (and thus browsers coming with Playwright) in my Robot tests in Docker, without a success. Getting similar kind of library issues but I have not got even Firefox to work in my image. Major part of the Dockerfile I am using is actually coming from your Dockerfile (Thanks @ppodgorsek!), but I have also some of my own stuff (for example a Node.js Express server, jest and TestCafe) that I want to include into the same image and that's why I wanted to make my own Dockerfile.

**Comment by izderadicka** [comment]
Same issue here

**Comment by dlabordus** [comment]
Same issue here. The message "no version information available" when running with the Chromium Browser. 
The Firefox browser has no problems.

**Comment by izderadicka** [comment]
I solved problems by creating my own image based on MS official image (which uses Ubuntu as base, because Ubuntu is the only officially supported distro for Playwright):

```Dockerfile
FROM mcr.microsoft.com/playwright:focal

ENV TZ=Europe/Prague

RUN pip3 install robotframework robotframework-browser && \
    rfbrowser init

RUN mkdir /tests && \
    chown pwuser /tests && \
    chmod 0777 /tests

USER pwuser
WORKDIR /tests
```

**Comment by ppodgorsek** [comment]
This issue was caused by the base image (Alpine) not providing the required packages. It has been fixed as part of issue #386.
